src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -310,19 +311,34 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 
 	private lazy var healthCertificateService: HealthCertificateService = HealthCertificateService(
 		store: store,
-		signatureVerifying: DCCSignatureVerifyingStub(error: .HC_COSE_NO_SIGN1),
+		signatureVerifying: dccSignatureVerificationService,
+		dscListProvider: dscListProvider,
 		client: client,
 		appConfiguration: appConfigurationProvider
 	)
 
+	private lazy var dccSignatureVerificationService: DCCSignatureVerifying = {
+		#if DEBUG
+		if isUITesting {
+			return DCCSignatureVerifyingStub(error: .HC_COSE_NO_SIGN1)
+		}
+		#endif
+
+		return DCCSignatureVerifyingStub(error: .HC_COSE_NO_SIGN1)
```

Comment:
I think i discovered the same in #3217: For me it feels some kind of wrong or suspect if we use in production code some sort of stub. Is this really a stub?

Reply:
The problem is that we do not have the production implementation yet, only the protocol and a stub. We need to replace it as soon as the actual implementation is finished.

Reply:
ahhh okay the detail I did not know. Then it is fine.